### DISCUSSION TITLE
Are there other Iree.zoo out there?

### DISCUSSION THREAD

**Discussion by GeorgeS2019** [iree-gd/iree.gd#83] [General]
https://github.com/iree-gd/iree.zoo/releases

It seems the collection is limited

**Comment by RechieKho** [comment]
Hello! Thanks for checking out iree.gd. In reality, that zoo is established a few weeks ago and not a lot of stuff is done to it 😳. The zoo is maintained by @Ughuuu I guess.

**Comment by Ughuuu** [comment]
The collection is limited indeed, if you want some things we can add it. If you want some things you can also clone the iree.zoo and build things urself.

The zoo has 2 actions. The multi action will have a max limit of 256 (as per GitHub actions).
The single one can be run and it build anything.

The need for it is because iree doesn't work directly with tensorflow files, they need to be built. Also every new release of iree we need to update the models.

I have built around 30 models, out of which only half actually built, and the rest resulted in errors.

The purpose of the zoo is to keep adding popular models that people might need so it's easier to use them with iree.gd
